src/go/configmanager/config_manager.go

```diff
@@ -45,24 +48,21 @@ var (
 					GCP metadata server will not be called to fetch access token, and
 					following flags will be ignored; --service_config_id, --service,
 					--rollout_strategy`)
-
-	// secured HTTP client calling service management service.
-	serviceConfigFetcherClient *http.Client
 )
 
 // Config Manager handles service configuration fetching and updating.
 // TODO(jilinxia): handles multi service name.
 type ConfigManager struct {
-	serviceName        string
-	serviceInfo        *configinfo.ServiceInfo
-	envoyConfigOptions options.ConfigGeneratorOptions
-	curRolloutID       string
-	curConfigID        string
+	serviceName               string
+	serviceInfo               *configinfo.ServiceInfo
+	envoyConfigOptions        options.ConfigGeneratorOptions
+	configIdServiceConfigFile string
```

Comment:
How about keep configId here?  if from fetcher,  update the config_id here too.

Reply:
Is there a specific reason for this Wayne? I think it makes sense to keep all service config related items decoupled from ConfigManager. I like the current implementation of placing `configId` in the `ServiceConfigFetcher`

Reply:
We need config_id from service_config_path flow too.  I just want two flows to store it in the same place.